### ISSUE TITLE
Scrolling is Awful

### ISSUE THREAD

**Issue by Quest79** [mienaiyami/yomikiru#391]
### Type of reader (if reader related)

ALL mostly vertical scroll 

### Description of the new feature / enhancement

Scrolling should be smooth. Not janky. Even firefox scrolls smoother than this. Huge impact on quality of reading relies on how smooth pages scroll. Feels like its scrolling at 5fps.

### Scenario when this would be used?

ALL

**Comment by mienaiyami** [comment]
Can you give some more details? When it is happening, in manga or epub? What is total chapter count in sidelist?


Also is hardware acceleration disabled? Try enabling it and see if there is any difference.

**Comment by raeka21** [comment]
I think the person who wrote this issue hadn't bothered with the scroll speed option in the reader setting, or that at the time no such option existed.

Better to just close this issue I recommend.

**Comment by mienaiyami** [comment]
Yeah, I know the issues are piling up — just haven’t had enough free time lately to sort through and manage them properly.
I even finished the new gallery view mode for Yomikiru a month ago but haven’t had time to clean up everything else before releasing.
Appreciate the reminder. I’ll try to go through and clean things up when I can.
